client/my-sites/domains/domain-management/edit/domain-types/mapped-domain-type.jsx

```diff
@@ -145,6 +155,28 @@ class MappedDomainType extends React.Component {
 		);
 	}
 
+	renderPendingGSuiteTosNotice() {
+		const { domain, selectedSite } = this.props;
+
+		if (
+			! hasPendingGSuiteUsers( domain ) ||
+			( ( ! this.props.isJetpackSite || this.props.isSiteAutomatedTransfer ) &&
+				! domain.pointsToWpcom ) ||
+			isExpiringSoon( domain, 30 )
+		) {
+			return null;
+		}
```

Comment:
That's just a suggestion but when conditions are beginning to be difficult to read I would usually break them into multiple `if` clauses. It's also possible to take advantage of destructuring to save a few characters here:

```javascript
const { domain, isJetpackSite, isSiteAutomatedTransfer, selectedSite } = this.props;

if ( ! hasPendingGSuiteUsers( domain ) || isExpiringSoon( domain, 30 ) ) {
  return null;
}

if ( ! domain.pointsToWpcom && ( ! isJetpackSite || isSiteAutomatedTransfer ) ) {
  return null;
}
```

Reply:
Sounds good :)

Reply:
Actually I'm going to leave it as is for now, especially since destructuring would shadow the imported functions.